src/pages/JobDescription/components/JobDescriptionItem.jsx

```diff
@@ -0,0 +1,36 @@
+import { Grid, ToggleButton } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+/**
+ * A component that displays a training block with course name, skills, progress, and description.
+ * @param {Object} props - The props object.
+ * @param {Object} props.JobDescription - The training object containing course name, skills, progress, description, and status.
+ * @param {string} props.mockedSkills - Prints the Skills in the toggles
+ * @returns {JSX.Element} the JobDescriptionPage Component.
+ */
+function JobDescriptionItem(props) {
+  return (
+    <Grid item xs={3} justifyContent={"center"}>
```

Comment:
Can you provide breakpoints, because I don't think this looks good: 
![image](https://github.com/SE2-WS23/Skillverse-Web/assets/37250473/dc43c087-0a78-4c83-bdd6-130384cd409f)